README.md

```diff
@@ -0,0 +1,13 @@
+# Scan-o-matic project converted
+
+1. Place all projects you wish to migrate in `/tmp/SoM/`
+
+**NOTE**: There's no way to migrate back from the
+python 3 version of projects to the old python 2.7 version
+and contents of files will be overwritten.
```

Comment:
I think this is all right, but I guess you could create an output folder and put converted projects there, for added safety.

Reply:
That would be highly annoying since the converter will not touch the various npy serialized objects nore the tiff image so the new projects would be useless. And if it would copy everything it would produce a ton of material copying all images.